lib/load-config.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+require = require('esm')(module); // eslint-disable-line no-global-assign
```

Comment:
Let's do `const requireEsm` though. We don't need to change any other imports.

Reply:
no problem, ✅